src/components/dialogs/NameRequestInvalidErrorDialog.vue

```diff
@@ -1,34 +1,35 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="namerequest-invalid-error-dialog">
     <v-card>
-      <v-card-title id="dialog-title" v-if="type === NameRequestStates.NOTFOUND">Name Request Not Found</v-card-title>
-      <v-card-title id="dialog-title" v-else>Name Request Invalid</v-card-title>
+      <v-card-title id="dialog-title">Invalid Name Request</v-card-title>
```

Comment:
This and below are simplifications to this dialog. I tested all cases (incl unit tests!) and this made the most sense to me for all cases.